src/helpers/fetches.js

```diff
@@ -70,6 +71,7 @@ export const fetchNormalizedServiceByName = async (client, serviceName) => {
 
     return normalizedTrigger.data
   } catch (e) {
-    throw new Error(`Can't find ${serviceName} : ${e}`)
+    log('error', `Can't find ${serviceName} : ${e}`)
+    return null
```

Comment:
Is it ok to return null here rather than an error (knowing that this is a function that is supposed to return the service) or is it better to return an error, and put conditions in the function that return null at the right time?

Reply:
It depends on the convention of your service. If it is expected that the service will sometimes not be installed, it makes to sense that a `null` expresses the fact that the service has not been found. Removing the need of a try/catch for the caller makes it easier to write the calling code, but the calling code won't be able to discriminate between a legitimate error (the service has not been found) and a (maybe) unlegitimate one (no internet). Since you are designing the function, it's you that has to make the call on what is the most convenient for the caller, and which trade-offs you are willing to make.